scripts/devtools/publish-release.js

```diff
@@ -82,7 +82,16 @@ async function publishToNPM() {
     // If so we might be resuming from a previous run.
     // We could infer this by comparing the build-info.json,
     // But for now the easiest way is just to ask if this is expected.
-    const info = await execRead(`npm view ${npmPackage}@${version}`);
+    const info = await execRead(`npm view ${npmPackage}@${version}`)
+      // For npm v8+ (node 16+) view command responds with 404 code, instead of 200 with empty response
```

Comment:
I tested with node v16.14.2 (npm v8.5.0) it still gives empty response instead of error.
You don't have to find the exact version tho. You can just say something like "early versions of npm view gives empty response, but older versions give 404 error. Catch the error to keep it consistent."

Reply:
> I tested with node v16.14.2 (npm v8.5.0) it still gives empty response instead of error. You don't have to find the exact version tho. You can just say something like "early versions of npm view gives empty response, but older versions give 404 error. Catch the error to keep it consistent."

Agh, my node v16.17.1 (npm 8.15.0) produces an errors

Anyways, thanks for suggestion, I've updated the comment